examples/zone-printer/README.md

```diff
@@ -0,0 +1,247 @@
+# Zone Printer
+
+Zone printer is an application that displays the
+[Google Cloud Platform (GCP) zone](https://cloud.google.com/compute/docs/regions-zones/)
+where the application is running. 
+
+# How does it work?
```

Comment:
How about moving this below?

Reply:
People like to understand what they are doing before going and running a few commands, IMO? At least I personally prefer that instead of running arbitrary commands on the Internet. People who don't care can always skip this section?

I am happy to move it if you really insist.